system_baseline/models.py

```diff
@@ -54,4 +58,40 @@ def to_json(self, withhold_facts=False):
         json_dict["updated"] = self.modified_on.isoformat() + "Z"
         if not withhold_facts:
             json_dict["baseline_facts"] = self.baseline_facts
+        if not withhold_system_ids:
+            json_dict["system_ids"] = self.mapped_system_ids()
         return json_dict
+
+    def add_mapped_system(self, system_id):
+        new_mapped_system = SystemBaselineMappedSystem(
+            system_id=system_id, account=self.account
+        )
+        self.mapped_systems.append(new_mapped_system)
+        db.session.add(new_mapped_system)
+
+    def remove_mapped_system(self, system_id):
+        system_id_removed = False
+        for mapped_system in self.mapped_systems:
+            if mapped_system.system_id == system_id:
+                self.mapped_systems.remove(mapped_system)
+                system_id_removed = True
+                break
+        if not system_id_removed:
+            # do we want to raise exception here?
```

Comment:
That's really a good question.

I guess that b/c we are trying to delete multiple records,
it should be ok to just silently pass already missing records.

Reply:
I think so, too.  However if this were something done in the db, it would raise an exception.  I think for now we can leave it as-is and not raise an exception.

Reply:
I changed it to raise ValueError exception.  I added a unit test for this as well.